packages/core/hooks/check-plan-gate.py

```
@@ -46,14 +51,16 @@ def _get_gate_path(data: dict) -> str:
 
 @safe_hook_execution
 def main() -> None:
-    data = json.load(sys.stdin)
+    data = read_hook_input()
 
     # Agent ツール以外は無視（後方互換のため "Task" も許容）
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)